s2.asm

```diff
@@ -176,7 +176,7 @@ ErrorTrap:
 ; loc_206:
 EntryPoint:
 	tst.l	(HW_Port_1_Control-1).l	; test ports A and B control
-	bne.s	PortA_Ok	; If so, branch.
+	bne.s	PortA_Ok	; If so, magically branch.
```

Comment:
Why magically? I believe this just checks if the IO ports have been initialised already, and skips the init process if so, nothing magic about it.